### ISSUE TITLE
Crash on attempt to call up Preferences

### ISSUE THREAD

**Issue by britram** [gnunn1/terminix#774]
Cloned master, built from source on current debian/jessie, Terminix appears to work well under standard Gnome 3 desktop until I choose Preferences... from the application menu, which yields the following crash:

```
% terminix
object.Error@(0): The function you are calling is not pressent in your version of GTK+.
----------------
??:? extern (C) void gtkc.Loader.Linker.unsupportedSymbol() [0xdff605]
??:? gtk.PopoverMenu.PopoverMenu gtk.PopoverMenu.PopoverMenu.__ctor() [0xc88395]
??:? gtk.PopoverMenu.PopoverMenu gx.terminix.prefeditor.titleeditor.TitleEditBox.createPopover(gx.terminix.prefeditor.titleeditor.TitleEditScope) [0xb8f1b4]
??:? void gx.terminix.prefeditor.titleeditor.TitleEditBox.createUI(gx.terminix.prefeditor.titleeditor.TitleEditScope) [0xb8ebf7]
??:? gx.terminix.prefeditor.titleeditor.TitleEditBox gx.terminix.prefeditor.titleeditor.TitleEditBox.__ctor(gtk.Entry.Entry, gx.terminix.prefeditor.titleeditor.TitleEditScope) [0xb8f4fc]
??:? gtk.Box.Box gx.terminix.prefeditor.titleeditor.createTitleEditHelper(gtk.Entry.Entry, gx.terminix.prefeditor.titleeditor.TitleEditScope) [0xb8eb01]
??:? void gx.terminix.prefeditor.prefdialog.AppearancePreferences.createUI() [0xb834e9]
??:? gx.terminix.prefeditor.prefdialog.AppearancePreferences gx.terminix.prefeditor.prefdialog.AppearancePreferences.__ctor(gio.Settings.Settings) [0xb83980]
??:? void gx.terminix.prefeditor.prefdialog.PreferenceDialog.createUI() [0xb7d4af]
??:? gx.terminix.prefeditor.prefdialog.PreferenceDialog gx.terminix.prefeditor.prefdialog.PreferenceDialog.__ctor(gtk.ApplicationWindow.ApplicationWindow) [0xb7f05a]
??:? void gx.terminix.application.Terminix.presentPreferences() [0xb3ceff]
??:? void gx.terminix.application.Terminix.onShowPreferences() [0xb3a225]
??:? _D2gx8terminix11application8Terminix14installAppMenuMFZ75__T12__dgliteral5TC4glib7Variant7VariantTC3gio12SimpleAction12SimpleActionZ12__dgliteral5MFC4glib7Variant7VariantC3gio12SimpleAction12SimpleActionZv [0xb3a13b]
??:? extern (C) void gio.SimpleAction.SimpleAction.callBackActivate(gtkc.giotypes.GSimpleAction*, gtkc.glibtypes.GVariant*, gio.SimpleAction.SimpleAction.OnActivateDelegateWrapper) [0xc0e445]
??:? g_closure_invoke [0xc2914244]
```

I suspect this is something silly with my environment (wrong library version somewhere?) but shouldn't this fail to build in that case?

(and many thanks for Terminix: it appears to be exactly what I was looking for presuming I can set the prefs up ;) ),

**Comment by gnunn1** [comment]
What version of GTK are you using, looks like I'm calling a newer API that isn't available in your version.

**Comment by gnunn1** [comment]
Are you in GTK 3.14 or earler as the method that is failing looks to be available since 3.16

**Comment by gnunn1** [comment]
Added a check for GTK 3.16
